clusterloader2/api/default.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"fmt"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+// SetDefaults set the default configuration parameters.
+func (conf *Config) SetDefaults() {
+	//To support deprecated "automanangedNamespaces" in testconfig
+	conf.AutomanagedNamespaces = 1
```

Comment:
I don't think it's needed. Below we default `namespace.number`, so that one will be used anyway..
I think we should remove it.

Reply:
done